terraform-provider-constellation/internal/provider/provider_test.go

```diff
@@ -34,7 +35,11 @@ var testAccProtoV6ProviderFactories = map[string]func() (tfprotov6.ProviderServe
 // acceptance testing when running under Bazel.
 func bazelSetTerraformBinaryPath(t *testing.T) {
 	if !runsUnderBazel() {
-		t.Skip("Skipping test as not running under Bazel.")
+		fmt.Println(
+			"Test is not runding under Bazel.\n" +
+				"Using Host Terraform binary for acceptance testing.\n" +
+				"Tests results may vary from the defaults (which run under Bazel).",
+		)
```

Comment:
the TF PATH is not necessary locally
```suggestion
		)
		return
```